src/components/Accordions/AccordionBoxItems.tsx

```diff
@@ -59,7 +59,7 @@ export default function AccordionBoxItems({
         key={uuid()}
         variant="splitted"
         selectionMode={selectionMode}
-        className="p-0 mb-4"
```

Comment:
because of this, the legend accordion (and possibly others) now doesn't have a bottom margin either, which looks weird. maybe this setting could be set via a prop?